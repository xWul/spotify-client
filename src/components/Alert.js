import React from 'react'
import styled from 'styled-components'

const Alert = styled.div`
  display: flex;
  text-align: center;
  font-size: 14px;
  border: 2px solid;
  border-color: #BA2B2B;
  background-color: #DE6045;
  color: #FFF;
  padding: 7px;
  margin: 5px;
  border-radius: 2px;
  text-align: center;
  line-height: normal;
  opacity: 1;
  transition: all 0.3s ease;
  justify-content: center;
`

export default function (props) {
  return (
    <Alert {...props} />
  )
}
