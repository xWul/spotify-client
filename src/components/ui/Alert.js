import React from 'react'
import styled from 'styled-components'

const Alert = styled.div`
  font-size: 14px;
  border: 2px solid;
  border-color: #BA2B2B;
  background-color: #DE6045;
  color: #FFF;
  width: 100%;
  padding: 7px;
  margin: 5px;
  border-radius: 2px;
  text-align: center;
  line-height: normal;
  opacity: 1;
  transition: all 0.3s ease;
`

export default function (props) {
  return (
    <Alert {...props} />
  )
}
