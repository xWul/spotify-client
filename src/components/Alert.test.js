import React from 'react'
import renderer from 'react-test-renderer'
import Alert from './Alert'

test('Alert renderer without crashing', () => {
  const component = renderer.create(
    <Alert>Testing</Alert>,
  )

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
