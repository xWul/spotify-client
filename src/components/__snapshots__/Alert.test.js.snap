// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renderer without crashing 1`] = `
<div
  className="sc-bdVaJa KsjYc"
>
  Testing
</div>
`;
